imports/client/components/CallSection.tsx

```diff
@@ -27,6 +27,7 @@ import MeteorUsers from "../../lib/models/MeteorUsers";
 import type { PeerType } from "../../lib/models/mediasoup/Peers";
 import mediasoupRemoteMutePeer from "../../methods/mediasoupRemoteMutePeer";
 import type { Action, CallState } from "../hooks/useCallState";
+import { Theme } from "../theme";
```

Comment:
this (and others) will want to be a type import.  Should be something `biome check --write` can fix.